utils/generic.py

```diff
@@ -1535,3 +1535,17 @@ def interval_length(interval: hl.Interval) -> int:
         )
     else:
         return interval.end.position - interval.start.position
+
+
+def rep_on_read(path: str, n_partitions: int) -> hl.MatrixTable:
+    """
+    Repartitions a MatrixTable on read
```

Comment:
It should be made clear when to use this function as opposed to `read` and `repartition` sequentially.